src/web/app/src/components/ThemeProvider.tsx

```diff
@@ -1,19 +1,17 @@
 import { createContext, useContext } from 'react';
 import { Theme } from '@material-ui/core/styles';
-
-import { ThemeName } from '../interfaces';
 import { lightTheme } from '../theme';
 
 type ThemeContextType = {
   theme: Theme;
-  themeName: ThemeName;
-  toggleTheme: () => void;
+  preferredTheme: String | undefined;
```

Comment:
Why do we need this to have the option to be undefined?

Reply:
I removed all possibilities that `preferredTheme` gets undefined, but I have a typescript error in IDE. Since lint is not complaining and the code compile has no problem, I will push without this `undefined`. Thanks for the feedback.